components/src/dynamo/trtllm/request_handlers/handler_base.py

```
@@ -905,6 +937,29 @@ async def _initiate_shutdown(self, error: Exception):
+        if fatal:
+            logging.error(
+                f"Fatal CUDA error in request {request_id} "
+                f"(caught as RequestError): {error_msg}",
+                exc_info=True,
+            )
+        else:
+            logging.warning(f"Request {request_id} error: {error_msg}")
+        try:
+            yield {"finish_reason": {"error": error_msg}, "token_ids": []}
+        except Exception:
+            pass  # Best effort
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)